apps/hash-frontend/src/shared/layout/layout-with-sidebar/account-entity-type-list.tsx

```diff
@@ -196,74 +158,61 @@ export const AccountEntityTypeList: FunctionComponent<
               alignItems="center"
               flex={1}
               mr={0.25}
-              pl={3.5}
+              pl={2}
               position="relative"
             >
-              {/*
-                Commented this out because the functionality is not present yet
-                ("View All Pages" screen hasn't been designed/built)
-
-                @todo uncomment when this has been done
-              */}
-
-              {/* <Link
-                href="/"
+              <Link
+                href="/types"
                 noLinkStyle
                 tabIndex={-1}
                 sx={{
                   mr: "auto",
                   flex: 1,
+                  opacity: searchVisible ? 0 : 1,
+                  transition: ({ transitions }) =>
+                    transitions.create("opacity"),
                 }}
               >
                 <Typography
                   variant="smallTextLabels"
                   sx={({ palette }) => ({
-                    fontWeight: 600,
+                    fontWeight: 500,
                     color: palette.gray[80],
+                    fontSize: 14,
+                    marginLeft: -1.5,
+                    px: 1.5,
+                    py: 0.5,
+                    borderRadius: "100px",
+                    ":hover": {
+                      color: palette.gray[90],
+                      background: palette.gray[15],
+                      "> svg": {
+                        color: palette.gray[90],
+                        marginLeft: 1.5,
+                      },
+                    },
                   })}
                 >
                   View All Types
+                  <ArrowRightIcon
+                    sx={{
+                      marginLeft: 0.75,
+                      fontSize: 10,
+                      color: ({ palette }) => palette.gray[80],
+                      transition: ({ transitions }) =>
+                        transitions.create(["color", "margin-left"]),
+                    }}
+                  />
                 </Typography>
-              </Link> */}
-
+              </Link>
               <SearchInput
                 searchVisible={searchVisible}
                 showSearchInput={() => setSearchVisible(true)}
                 hideSearchInput={() => setSearchVisible(false)}
```

Comment:
Appreciate this is pre-existing but the button is marked 'Clear search' and currently doesn't
```suggestion
                hideSearchInput={() => {
                  setSearchQuery("");
                  setSearchVisible(false);
                }}
```